README.md

```diff
@@ -36,25 +36,25 @@ Response function(Request request) => Response.ok('Hello, World!');
 
 To:
 
-```sh
-curl http://my-url
+```shell
+$ curl https://example.com
```

Comment:
Don't include $ in bash scripts. They aren't copy pastable.

Reply:
Removed this one. Should chat w/ @subfuzion if we want to clean up the rest in the repo